src/shared/components/palette/complex/read-complex-field.html

```diff
@@ -4,6 +4,11 @@
     [caseField]="caseField"
     [context]="context"
   ></ccd-read-complex-field-raw>
+  <ccd-read-complex-field-new-table
```

Comment:
I guess the ...-new-table... came from the fact we already have ...-field-table... suffix.

Reply:
Can we make the already existing ccd-read-complex-field-table accommodate the "new table" concept?

Reply:
I would keep this separate new collections field is going get more complicated